sdk/spring/azure-spring-boot/README.md

```diff
@@ -1,21 +1,5 @@
-#Azure Spring Boot client library for Java
-
-## Key concepts
-This project provides auto-configuration for the following Azure services:
-
-- [Azure Active Directory](../azure-spring-boot-starter-active-directory)
-- [Azure Active Directory B2C](../azure-spring-boot-starter-active-directory-b2c)
-- [Cosmos DB SQL API](../azure-spring-boot-starter-cosmosdb)
-- [Key Vault](../azure-spring-boot-starter-keyvault-secrets)
-- [Media Service](../azure-spring-boot-starter-mediaservices)
-- [Service Bus](../azure-spring-boot-starter-servicebus)
-- [Storage](../azure-spring-boot-starter-storage)
-
-This module also provides the ability to automatically inject credentials from Cloud Foundry into your
-applications consuming Azure services. It does this by reading the `VCAP_SERVICES` environment
-variable and setting the appropriate properties used by auto-configuration code.
-
-For details, please see sample code in the [azure-spring-boot-sample-cloud-foundry](../azure-spring-boot-samples/azure-spring-boot-sample-cloud-foundry) 
+# Azure Spring Boot client library for Java
+This repo is for Spring Boot Starters of Azure services. It helps Spring Boot developers to adopt Azure services.
 
 ## Getting started
 To start a new project using Azure, go on [start.spring.io](https://start.spring.io) and select "Azure
```

Comment:
Let's use another starter as the example here, we've deprecated the starter for service bus 😸